packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/Model.java

```diff
@@ -32,6 +31,7 @@ public class Model extends TerariumAsset implements SupportAdditionalProperties,
 
 	private Map<String, Object> model;
 
+	@TSOptional
 	private String modelCard;
```

Comment:
the `modelCard` is part of the `metadata` property, we do not need a new property.